package time;

public class TimeConverter {
    public void hoursToMinutes(double hours) {
        System.out.println("Hours to Minutes = " + (hours * 60));
    }

    public void minutesToHours(double minutes) {
        System.out.println("Minutes to Hours = " + (minutes / 60));
    }

    public void hoursToSeconds(double hours) {
        System.out.println("Hours to Seconds = " + (hours * 3600));
    }

    public void secondsToHours(double seconds) {
        System.out.println("Seconds to Hours = " + (seconds / 3600));
    }
}
